frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -272,6 +282,7 @@ local long_press_action = {
     {_("Ask with popup dialog"), "ask"},
     {_("Do nothing"), "nothing"},
     {_("Highlight"), "highlight"},
+    {_("Select / highlight"), "select"},
```

Comment:
`Two-steps highlight` ?
`Select + highlight` ? (`/` looks like an alternative)

Reply:
> `/` looks like an alternative

Agree. Maybe `Select and highlight` or `Highlight selection`.